Bookish.DataAccess/Services/BookishService.cs

```diff
@@ -104,6 +104,25 @@ public void AddBook(string title, string author, string isbn, int totalCopies)
             dbConnection.Execute(query, new { Title = title, Author = author, ISBN = isbn });
         }
 
+        public IEnumerable<BookCopy> GetTotalCopies(string isbn)
+        {
+            var query =
+                @"SELECT
+                    Books.Title AS Title,
+                    Books.Author AS Author,
+                    BookCopies.CopyID AS CopyID,
+                    AspNetUsers.UserName AS Username,
+                    Loans.DueDate AS DueDate
+                FROM
+                    BookCopies
```

Comment:
Does this query work? You're trying to access values from the Books table in your select but you're not joining on the Books table 🙂

Reply:
I stopped implementing this method after asking you about the question about redundant data and forgot to remove it.